src/test/java/test/CalculatorTest.java

```diff
@@ -0,0 +1,87 @@
+package test;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import step1.Calculator;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class CalculatorTest {
+    int a = 0;
+    int b = 0;
+    int c = 0;
+    int d = 0;
+    String expression = "";
+    @BeforeEach
+    void setUp() {
+        a = 10;
+        b = 15;
+        c = 20;
+        d = 40;
+        expression = "1 - 3 + 4 * 10 / 7";
+    }
+
+    //더하기
+    private int _sum(int a , int b){
+        return a+b;
+    }
+    //뺄셈
+    private int _minus(int a , int b){
+        return a-b;
+    }
+    //곱셈
+    private int _multiply(int a , int b){
+        return a*b;
+    }
+    //나눗셈
+    private int _divide(int a , int b){
+        return (int)(a/b);
+    }
+
+    @Test
+    @DisplayName("더하기 Test")
+    public void sum(){
+        assertThat(a+b).isEqualTo(25);
+    }
```

Comment:
해당 테스트의 주체는 Calculator라고 생각합니다 ㅎㅎ
Calculator를 생성하여 Calculator를 통해 계산을 테스트해보시면 어떨까요 :) ?
```suggestion
    @Test
    @DisplayName("더하기 Test")
    public void sum(){
        final String input = "20 + 5";
        final Calculator calculator = new CalculatorI();
        final int expected = 25;
        
        final int actual = calculator.execute(input);
        
        assertThat(actual).isEqualTo(expected);
    }
```